### ISSUE TITLE
Filter request

### ISSUE THREAD

**Issue by OGContent** [Minehut/Meta#584]
### Filter request

**Checklist**
- [x ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ x] I have ensured the phrase is not properly filtered on the platform

**Platform**

ingame 
**Should the phrase be filtered or unfiltered?**

unfiltered because my server "Cropied69" i cant ad it at all
**Phrase**

```69

```

**Comment by quick007YT** [comment]
Uh whats supposed to be here? 
![image](https://user-images.githubusercontent.com/69743171/114252457-2746f480-995a-11eb-80d1-7ed3e5ef9c4f.png)

**Comment by quick007YT** [comment]
ah I saw it on discord, fine with unblocking 69 yea

**Comment by BennyDoesStuff** [comment]
No longer relevant.
